src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -0,0 +1,124 @@
+// React Imports
+import React, { useContext } from 'react';
+// Inrupt Library Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Utility Imports
+import { SOLID_IDENTITY_PROVIDER, checkContainerPermission, runNotification } from '../../utils';
+// Custom Hook Imports
+import { useField, useStatusNotification } from '../../hooks';
+// Context Imports
+import { SelectUserContext } from '../../contexts';
+// Component Imports
+import FormSection from './FormSection';
+
+/**
+ * CheckAclPermsDocContainerForm Component - Component that generates the form
+ * for checking if ACL permissions to another user's Documents container are
+ * available
+ *
+ * @memberof Forms
+ * @name CheckAclPermsDocContainerForm
+ */
+
+const CheckAclPermsDocContainerForm = () => {
+  const { session } = useSession();
+  const { state, dispatch } = useStatusNotification();
+  const { clearValue: clearUrl, ...user } = useField('text');
```

Comment:
The useField custom hook will likely be replaced by useState once more components transitions to MUI.